examples/battery_monitor/gcc-Revo-toolchain.cmake

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.10)
+
+message("LOADING STM32F4 TOOLCHAIN")
+# Generic is used for sytems that can't run CMake
+SET(CMAKE_SYSTEM_NAME Generic)
```

Comment:
what is this?

What systems do we have that can't run CMake?

Reply:
The Revo. System name indicates what system type it is run for: windows, Linux, etc. Generic is used for systems that don't support Cmake themselves, such as stm32.